### ISSUE TITLE
jsrsasign dependency

### ISSUE THREAD

**Issue by Ibro** [damienbod/angular-auth-oidc-client#11]
I think the library should handle dependency on jsrsasign, so the developer doesn't have to do it 'manually'. Should figure out a proper way to do it.

**Comment by damienbod** [comment]
@Ibro thanks, this makes sense. First time I tried this, the  jsrsasign npm package was missing dependencies which this package requires. I need to figure out which packages are required from jsrsasign. Thanks for the suggestion. 

If you have time, maybe you could create a pull request to improve this.

Greetings Damien

**Comment by robisim74** [comment]
@Ibro Thanks for the suggestion.

@damienbod Finally, I think, I did the trick: https://github.com/damienbod/angular-auth-oidc-client/tree/jsrsasign_integration

In the version on the branch, `jsrsasign` is a dependency of the library, and I found a way to import it also without its definition file: but when you use its functions, you will have to add them to _rollup.config_.
In this way, devs must not install or import it in any way. I could only test it on the sample app. Try it yourself (remember to remove the import script from index.html) and see if you like it.
Greetings
